diagrams/basic_jbi_diagram.md

```diff
@@ -0,0 +1,14 @@
+``` mermaid
+graph TD
+    subgraph bugzilla services
+        A[Bugzilla] -.-|bugzilla event| B[(Webhook Queue)]
+        B --- C[Webhook Push Service]
+    end
+    D --> |create/update/delete issue| E[Jira]
+    D<-->|Read Bug| A
+    D -->|update see_also| A
+    subgraph jira-bugzilla-integration
+        C -.->|webhook event| D{JBI}
```

Comment:
Out of curiosity: why does this arrow have a dotted line? 
Also, shouldn't it have a verb too? (eg. `post /bugzilla_webhook` or `post webhook event`)

Reply:
I used dotted lines for async operations; should I make them all solid lines?